### ISSUE TITLE
FileNotFoundError: [Errno 2] No such file or directory: './data/ShapeNet/data/ShapeNetTrain/04256520_9c1b448ec62cb9fb36dd029536673b0b_21.dat'

### ISSUE THREAD

**Issue by banannakn** [zouchuhang/Silhouette-Guided-3D#1]
I downloaded ShapeNet data and unzipped it but there is no folder "./data/ShapeNetTrain" included. I am not able to run "python train.py" and I can't find the dat-file anywhere. Any recommendations what to do or what went wrong?

**Comment by zouchuhang** [comment]
@banannakn You need to unzip all the downloaded .7z file to get the .dat file

**Comment by banannakn** [comment]
Yes, this is what I did. But the problem is that there is no folder "ShapeNetTrain", not even the upper folder "data", but I created them. Next problem, I need to put the ".dat files" in the folder "ShapeNetTrain" (it is written otherwise in the manual you wrote). That's what I did but there are still some ".dat files" missing, I just got them from number 0 to number 4. And the error says there should exist ".dat files" up to number 21 (the last two numbers of the".dat file" name) and I can't find them anywhere

**Comment by zouchuhang** [comment]
@banannakn Some reference for data downloading from the Pixel2Mesh website: https://github.com/nywang16/Pixel2Mesh/issues/12#issuecomment-437219435
https://github.com/nywang16/Pixel2Mesh/issues/42
https://github.com/nywang16/Pixel2Mesh/issues/5
I use the training data from Pixel2Mesh, you can check for more details.

**Comment by ghost** [comment]
I cannot decompress the 7z files. There seems to be error
`ERROR: Unknown error -2147024891`
`Archives with Errors: 1`

**Comment by zouchuhang** [comment]
@rangchu Thanks for your interests to our paper. I followed this link to download the ShapeNet dataset with generated GT point clouds: https://github.com/nywang16/Pixel2Mesh#dataset. I do not have the data saved in my current local machine so I cannot provide you my copies of the dataset. You can follow the dataset download step in my linked Pixel2Mesh git repo. If you have any questions (e.g. unzipping the 7z files), I suggest you post a question directly in the Pixel2Mesh git repo.

**Comment by star-cold** [comment]
> I downloaded ShapeNet data and unzipped it but there is no folder "./data/ShapeNetTrain" included. I am not able to run "python train.py" and I can't find the dat-file anywhere. Any recommendations what to do or what went wrong?

Have you solved the missing data problem? I also only find 00.dat to 04.dat，not more. Thank you!

**Comment by holawx** [comment]
I am confronted with the same problem that the ShapeNet datasets only contains the rendering 00.dat to 04.dat

**Comment by zouchuhang** [comment]
@holawx the rendered images are optional requirements as shown in the Readme file: https://github.com/zouchuhang/Silhouette-Guided-3D#download-data-and-pre-trained-model
I don't think this will affect you to run the existing code.
